src/Renderer/LayeredMaterial.js

```diff
@@ -172,14 +171,6 @@ LayeredMaterial.prototype.dispose = function dispose() {
             }
         }
     }
-
-    var jT = new JavaTools();
-
-    jT.freeArray(this.textures[0]);
```

Comment:
@peppsac actually, when they are fields, they might never go out of scope, so it might be necessary to assign null, undefined, or better, an empty array to deallocate the previous one.

@gchoqueux I didn't do it here, because it was working before, and I was afraid of the side-effect if I did fix it. ("if it ain't broken, don't fix it"). I wanted to stay iso with previous *real* behaviour. It would always be time to optimize later if we feel we need it.